docs/v4/mini-faq.md

```diff
@@ -0,0 +1,198 @@
+# Mini-FAQ
+
+@jesus-seijas-sp collected a mini FAQ for some v4 topics:
+
+Hi,
+
+I will put here a FAQ with the links to different interesting parts to documentation or examples. If there is something that you want to cover that is not here, just ask, and I will update the documentation with it. Also, there are 1800 unit tests that can help to understand classes and functions that are not intended to be the API for the developer (from node-nlp the API for the developer is intented to be the class NlpManager).
+
+**- Where do I find an example of use of v4?** https://github.com/axa-group/nlp.js#example-of-use
+
+**- But this example seems to be of the v3...** The version 4 is splitted into different smaller packages, but the https://www.npmjs.com/package/node-nlp package use tose smaller packages to build a version so retrocompatible with v3 as we can
+
+**- But I don't find the NluManager in this package...** You don't need it, you have the NlpManager that inside is able to handle the NLU, NLG, Language guesser....
```

Comment:
Also can be commented that if you just want to classify, you can use it directly the @nlpjs/nlu package
Here is an example:
https://github.com/axa-group/nlp.js/blob/master/examples/15-nlu/05-nlu-manager.js